examples/source/1.components/amp-story-interactive-poll/index.html

```diff
@@ -192,7 +196,11 @@
 </html>
 
 <!--
-        # More resources
+  ## Accessibility considerations
 
-        If you want to learn more about all the interactive experiences for Web Stories, head over to the [documentation](/documentation/components/amp-story-interactive?format=stories) or check out the guide on how to make a [story with quizzes](/documentation/examples/components/amp-story-interactive-quiz/?format=stories) instead.
-      -->
+  Currently, this component lacks keyboard and screen reader accessibility.
```

Comment:
Can we add a link to https://github.com/ampproject/amphtml/issues/32751?  That should hopefully get resolved soon, but good to have visibility.

Reply:
Do you want @TetraLogicalHelpdesk to make the suggested changes, or will you/the team do these additions before merging?

Reply:
Currently working on the PR to make it accessible, so no need to add it to the documentation